assets/scripts/collection.js

```diff
@@ -72,11 +72,22 @@ const createDOM = (collection) => {
     collection.dom = tableDOM;
     collection.head = theadDOM;
     collection.body = tbodyDOM;
+
+    const emptyDOM = document.createElement('td');
+    emptyDOM.colSpan = 7;
+    emptyDOM.style.textAlign = 'center';
+    emptyDOM.style.padding = '10px';
+    emptyDOM.innerHTML = 'No schematics found. You can sign up and upload your own schematics.';
```

Comment:
The placeholder text should be set by option so that it can be customized for the other tables as well.

Reply:
So?